platform/security/sts/security-sts-ldaplogin/src/main/java/ddf/ldap/ldaplogin/LdapLoginConfig.java

```diff
@@ -121,10 +129,34 @@ public void update(Map<String, ?> props) {
         ldapConnectionPool.close();
       }
 
-      LDAPConnectionFactory ldapConnectionFactory =
-          createLdapConnectionFactory(
-              (String) ldapProperties.get(LDAP_URL),
-              Boolean.parseBoolean((String) ldapProperties.get(START_TLS)));
+      ConnectionFactory ldapConnectionFactory = null;
+
+      List<String> urls = new ArrayList<>();
```

Comment:
✏️ This block of code is repeated [here](https://github.com/codice/ddf/blob/e114088676c415ac1e4907d5b3c98c75ca134dab/platform/security/sts/security-sts-ldapclaimshandler/src/main/java/ddf/security/sts/claimsHandler/ClaimsHandlerManager.java#L119). Consider making a helper class for similar functionality

Reply:
While this is true, the two places that this is duplicated in are already in separate bundles and sharing code between them would require a separate third bundle or a dependency between these two.  I don't feel that this is significant enough to warrant a whole new bundle or to create an unnecessary dependency linkage between these two bundles.